### PR TITLE
Reflex Items now all start with max level ability.

### PR DESCRIPTION
**Pull request by ThatMusicWriter**
Special thanks to Chris for pulling me into the Discord and to Gecko for explaining it in small words. Love you guys. <3

NOTE: Still too many commits, but choose your battles, as they say.

### CODE REVIEW

File: game/resource/English/ability/items/tooltip_reflection_shard.txt [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
-"DOTA_Tooltip_Ability_item_reflection_shard_1_bonus_mana_regen"                           "+$mana_regen"
 
-"DOTA_Tooltip_Ability_item_reflection_shard_2"                                            "Reflection Shard"
-"DOTA_Tooltip_Ability_item_recipe_reflection_shard_2"                                     "Reflection Shard Recipe"
-"DOTA_Tooltip_Ability_item_reflection_shard_2_Description"                                "<h1>Active: Counter</h1> Grants %duration% seconds of invulnerability and returns targeted spells."
-"DOTA_Tooltip_Ability_item_reflection_shard_2_Lore"                                       "A chromatic shard from another dimension."
-"DOTA_Tooltip_Ability_item_reflection_shard_2_bonus_all_stats"                            "+$all"
-"DOTA_Tooltip_Ability_item_reflection_shard_2_bonus_health_regen"                         "+$hp_regen"
-"DOTA_Tooltip_Ability_item_reflection_shard_2_bonus_mana_regen"                           "+$mana_regen"
+"DOTA_Tooltip_Ability_item_reactive"                                             "Invincibility Shard"
```
Comment (imaGecko) [inline game/resource/English/ability/items/tooltip_reflection_shard.txt]: should be "DOTA_Tooltip_Ability_item_reflection_shard_1"

**Comment by l34Um1** [comment]
You should do this as a new branch, new PR. So it doesn't have all the cave rune stuff and such.

**Comment by ThatMusicWriter** [comment]
I accidentally did all the cave rune stuff in master, and am still working on cleaning up the commits; they aren't in the changes I made here, and I'll remove the invalid commits when I have time next week.

**Comment by ThatMusicWriter** [comment]
Special Note: In order to prevent player confusion I am attempting to rename the level 1 Reflection Shard from "Invincibility Shard" to "Minor Reflection Shard," as the invincibility effect has been replaced by the reflection.

**Comment by chrisinajar** [comment]
This wont work and your commit history is insane....
See if you can get a nice clean starting branch again, and also make sure you execute the changes locally to make sure they work.

I'm going to close this so you can have a clean starting point better.

**Comment by imaGecko** [comment]
Cleaning up your commit history is as simple as resetting your master branch and creating a new branch from it. You can also copy your changes to desktop before creating a new branch and just paste them into the new branch for easy changes

**Comment by ThatMusicWriter** [comment]
Did it last night. files took time to update

**Pull request closed without merging**